tasks/run-matlab-tests/v1/task.json

```diff
@@ -74,7 +46,7 @@
             "defaultValue": "",
             "groupName": "filterTests",
             "required": false,
-            "helpMarkDown": "Locations of the folders used to select test suite elements, relative to the project root folder. To create a test suite, the task uses only the tests in the specified folders and their subfolders. You can specify multiple folders using a colon-separated or a semicolon-separated list."
+            "helpMarkDown": "Location of the folders used to select test suite elements, relative to the project root folder. To create a test suite, MATLAB uses only the tests in the specified folder and its subfolders. You can specify multiple folders using a colon-separated or a semicolon-separated list."
```

Comment:
Can we shorten by removing this part?

To create a test suite, MATLAB uses only the tests in the specified folder and its subfolders.

I am hoping the user refers to the doc for more info.